js/main.js

```diff
@@ -0,0 +1,2 @@
+// var $searchForm = document.querySelector('#search-form');
+// $searchForm.addEventListener('submit', handleSubmit);
```

Comment:
Where is your JavaScript? This feature should include making searches and getting data, you just won't show it to the user yet. Without any JavaScript, this feature is literally "User can see homepage" 🤮